test/rihanna/job_dispatcher_test.exs

```diff
@@ -196,6 +197,32 @@ defmodule Rihanna.JobDispatcherTest do
     end
   end
 
+  describe "handle_info/2 with reenqueued job" do
+    setup do
+      {:ok, dispatcher} =
+        Rihanna.JobDispatcher.start_link([db: Application.fetch_env!(:rihanna, :postgrex)], [])
+
+      {:ok, %{dispatcher: dispatcher}}
+    end
+
+    test "sets due_at on the job", %{pg: pg} do
+      now =
```

Comment:
Giant hack to support elixir without `DateTime.add/3`.

Reply:
Which version was this introduced in?

Can we at least move the logic into a helper function?

Reply:
Seems to have been introduced in 1.8+

Reply:
I found that `scheduled_job_test.exs` defined `due_in`. Moving that to `support/test_helper.exs` and re-using it.